jpo-ode-core/src/main/java/us/dot/its/jpo/ode/model/OdeSsmPayload.java

```diff
@@ -1,27 +1,30 @@
 package us.dot.its.jpo.ode.model;
 
-import com.fasterxml.jackson.annotation.*;
-
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import us.dot.its.jpo.ode.plugin.j2735.J2735SSM;
 
-public class OdeSsmPayload extends OdeMsgPayload {
+/**
+ * Represents the payload of an SSM message.
+ */
+public class OdeSsmPayload extends OdeMsgPayload<OdeObject> {
```

Comment:
Updating classes that inherit OdeMsgPayload to now explicitly provide the OdeObject class as the type. This used to be the default, but had to change to allow for using ASN POJOs as part of the payload. Other check style changes were also applied.